src/useTubular.ts

```diff
@@ -3,18 +3,18 @@ import * as ReactDom from 'react-dom';
 import {
     ColumnModel,
     CompareOperators,
+    DataGridStorage,
     GridRequest,
     GridResponse,
-    DataGridStorage,
-    TubularHttpClientAbstract,
     LocalStorage,
     NullStorage,
     sortColumnArray,
+    TubularHttpClientAbstract,
```

Comment:
This import sorting is from the vscode setting
```
"editor.codeActionsOnSave": {
        "source.organizeImports": true
    },
```

Which I believe comes from the ts compiler

Reply:
Yeah, but the error with lint might be not related to this one.